src/parsers/transactions.ts

```diff
@@ -290,9 +290,14 @@ const overrideTradeRefund = (txn: ZerionTransaction): ZerionTransaction => {
   };
 };
 
+const overrideAssetName = (name: string): string => {
+  return name.replace('Stablecoin', '').trim();
```

Comment:
Covalent likes adding `Stablecoin ` prefix. Zerion is not doing this

Reply:
Do this in the backend instead?

Reply:
I prefer not. I don't know what will be the ultimate solution here and keeping those small sanitations on mobile side is not problematic I guess. Do you have any severe reasons for moving this part to backend side?

Reply:
I'm also okay for just ignoring that. This is now a big deal if that will be Dai Stablecoin for L2